JellyfishClient/module.md

```diff
@@ -1,3 +1,5 @@
 # Module JellyfishClient
 
-## Overview
\ No newline at end of file
+# Package com.jellyfishdev.jellyfishclient.webrtc
+
+See [membrane-webrtc-android docs](https://jellyfish-dev.github.io/membrane-webrtc-android/) for API documentation.
```

Comment:
I'm confused now, those are docs for webrtc, not jellyfish? Is it intentional?

Reply:
Yes, those docs are there to provide an quick and easy way for jellyfish client user to understand how the webrtc functions that we're exposing through type aliases work. There is no easy way of exporting docs for types I made aliases of, and copy pasting them seems like a huge mistake, so I figured that this is the best solution